components/Alerts/alerts.css

```diff
@@ -0,0 +1,39 @@
+.alert-msg {
+    display: flex;
+    justify-content: left;
+    height: 57px;
+    width: 522px;
```

Comment:
suggestion: try to give use REM to make it resp rather ussing px

Reply:
okk thanks for your suggestion.